Appraisals

```diff
@@ -0,0 +1,24 @@
+appraise 'rails-4.1' do
+  gem 'actionpack', '~> 4.1.0'
+  gem 'activesupport', '~> 4.1.0'
+end
+
+appraise 'rails-4.2' do
+  gem 'actionpack', '~> 4.2.0'
+  gem 'activesupport', '~> 4.2.0'
+end
+
+appraise 'rails-5.0' do
+  gem 'actionpack', '~> 5.0.0'
+  gem 'activesupport', '~> 5.0.0'
+end
+
+appraise 'rails-5.1' do
+  gem 'actionpack', '~> 5.1.0'
+  gem 'activesupport', '~> 5.1.0'
+end
+
+appraise 'rails-5.2' do
+  gem 'actionpack', '~> 5.2.0.beta2'
+  gem 'activesupport', '~> 5.2.0.beta2'
```

Comment:
'~> 5.2.x'

Reply:
Thanks! Did not realize you could specify this way and pick up the beta.